core-bundle/src/Image/Studio/Figure.php

```diff
@@ -203,7 +202,7 @@ public function getOptions(): array
      *       when using Twig templates! Instead, add this object to your
      *       template's context and directly access the specific data you need.
      *
-     * @param string|array|null $margin              Set margins that will compose the inline CSS for the "margin" key
+     * @param string|array|null $margin              Set margins that will compose the inline CSS for the "margin" key. Deprecated, does not have any effect!
```

Comment:
Why do we keep the parameter at all?

Reply:
See https://github.com/contao/contao/pull/4788/files/9e54791c5f7a628cd7986e69730c99da139a0c97#r888876972
@ausi @m-vo